e2e/app.po.ts

```diff
@@ -217,25 +216,29 @@ export class AppPage {
       await new GithubLogin().authorizeApp();
     }
 
-    await browser.wait(ExpectedConditions.presenceOf(this.rootElement), 30 * 1000,
-    'syndesis root element - assuming we are already logged in');
+    await browser.wait(
+      ExpectedConditions.presenceOf(this.rootElement),
+      30 * 1000,
+      'syndesis root element - assuming we are already logged in'
+    );
 
     browser.waitForAngularEnabled(true);
     return this.goToUrl(AppPage.baseurl);
   }
 
-
   /**
    * Hook into browser and fetch config.json
    * @returns {any} config.json used in syndesis app
    */
   getSettings(): P<any> {
     // jquery is invoked in the context of the browser
-    return browser.driver.executeAsyncScript((callback) => {
-      jQuery.get('/config.json', function (data) {
-        callback(data);
-      });
-    }).then(jsonString => JSON.parse(<string> jsonString));
+    return browser.driver
+      .executeAsyncScript(callback => {
+        jQuery.get('/config.json', function(data) {
+          callback(data);
+        });
+      })
+      .then(jsonString => JSON.parse(<string>jsonString));
```

Comment:
have you tried `yarn lint` after this reformat? I recall some complains in similar situations like this one